java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -92,4 +92,18 @@ public void lastsave() {
         var response = clusterClient.exec(new ClusterTransaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    // TODO: Enable when https://github.com/amazon-contributing/redis-rs/pull/138 is merged.
+    // @Test
+    // @SneakyThrows
+    // public void objectFreq() {
+    //    String objectFreqKey = "key";
+    //    ClusterTransaction transaction = new ClusterTransaction();
+    //    clusterClient.configSet(Map.of("maxmemory-policy", "allkeys-lfu"), ALL_NODES).get();
+    //    transaction.set(objectFreqKey, "");
+    //    transaction.objectFreq(objectFreqKey);
+    //    var response = clusterClient.exec(transaction).get();
+    //    assertEquals(OK, response[0]);
+    //    assertTrue((long) response[1] >= 0L);
```

Comment:
I think response[1] will be OK and response[2] should be >= 0L, like you have in TransactionTests.java. Since response[1] corresponds to the `set` call

Reply:
I'm pretty sure there is no response[2] because only the SET and OBJECT FREQ commands are part of the transaction.

Reply:
isn't objectFreq also part of it?

Reply:
This one isn't doing a transaction.configSet, so it only does SET and OBJECT FREQ in the transaction. I can change it to be consistent with the other test though.